server/views/resources/how-nonprofit-projects-work.jade

```diff
@@ -0,0 +1,47 @@
+            <h3>Your first Nonprofit Project</h3>
+            <p>We generally scope Nonprofit Projects to 200 hours, or about 10 weeks at 20 hours per week of development per camper. These are only rough estimates. You’ll only work on one project at a time.</p>
+            <p>You will choose another camper to partner with on this projects. You should choose your pair based on:</p>
+            <p>Your estimated time commitments (10, 20 or 40 hours per week) Your time zones (Will you be able to pair program together?) Prior coding experience (we’d like both campers to be able to contribute equally) Do not take age or gender into account. This will provide you with valuable experience in meshing with diverse teams, which is a reality of the contemporary workplace.</p>
+            <p>Then, you’ll email <a href="mailto:team@freecodecamp.com">team@freecodecamp.com</a> with:</p>
+            <ol>
+                  <li>Names</li>
+                  <li>Contact information</li>
+                  <li>Links to each of your Code Portfolios</li>
+                  <li>Timezones</li>
+                  <li>Hours pledged per week</li>
+            </ol>
+            <p>We’ll send you a list of 3 nonprofit projects that need your help, and together you will choose one to begin work on. While you do this, our team will code review each of your Code Portfolios to ensure all user stories have been completed, and that there are no signs of academic dishonesty.</p>
+            <h3>Beginning the Project</h3>
+            <p>We’ll set an initial meeting with representatives from Free Code Camp, the two campers, and the stakeholder. If the stakeholder and both campers shows up promptly, and seem enthusiastic and professional, we’ll start the project. This lengthy process serves an important purpose: it reduces the likelihood that any of our campers or stakeholders will waste their precious time.</p>
+            <h3>Nonprofit Stakeholders</h3>
+            <p>Each nonprofit project was submitted by a nonprofit. A representative from this nonprofit has agreed to serve as a “stakeholder” - an authorative person who understands the organization and its needs for this particular project.</p>
+            <p>Stakeholders have a deep understanding of their organizations’ needs. Campers will work with them to figure out the best solutions to these needs.</p>
+            <p>When you and your pair partner first speak with your nonprofit stakeholder, you’ll:</p>
+            <p>talk at length to better understand their needs. create a new Trello board and use it to prioritize what needs to be built. and establish deadlines based on your weekly time commitment, and how long you think each task will take. Ideally, we’ll scope each project to be completed in 10 sprints. It’s notoriously difficult to estimate how long building software projects will take, so feel free to ask our volunteer team for help.</p>
```

Comment:
This paragraph is missing proper capitalization and has several grammar errors.  Periods should be commas?